src/geom/stroke.rs

```diff
@@ -1,19 +1,86 @@
 use super::*;
 
+/// A the line cap of a stroke
```

Comment:
```suggestion
/// The line cap of a stroke.
```
same below

Reply:
also, can you reorder these types from general to specific, with stroke being more general and linecap being more specific.